OrbitQt/ConnectToStadiaWidget.cpp

```diff
@@ -106,6 +106,7 @@ void ConnectToStadiaWidget::Start(SshConnectionArtifacts* ssh_connection_artifac
     service_deploy_manager_ = std::move(connection->service_deploy_manager_);
     grpc_channel_ = std::move(connection->grpc_channel_);
     state_machine_.setInitialState(&s_connected_);
+    ReloadInstances();
```

Comment:
This is probably related to a bug that I found - when you press "End Session", the list of instances is not loaded. You're still connected to your current instance, but if you press "Disconnect", you end up with an empty instance list and need to reload that list manually first.

Reply:
I just re-tested this - the bug is fixed, but it's still not ideal because the progress bar does not show up. So if you press "disconnect" right after the window shows up, Orbit is still loading the list of instances, but doesn't show any indication of this. I think this isn't a breaking issue for the merge, but we should look at this in the future.

Reply:
Actually, I've changed this now - s_instance_selected_ will now trigger re-loading of the instances when the state is entered and the table is empty. @beckerhe PTAL

Reply:
Oh you're right, this should be a separate PR, I completely missed that this wasn't introduced in this PR. If you could take care of splitting it up that'd be great.

Reply:
Great. I will take care of it.